packages/config-utils/proxy.js

```diff
@@ -31,7 +31,8 @@ module.exports = ({
     useCloud = false,
     target = '',
     keycloakUri = '',
-    registry = []
+    registry = [],
+    isChrome = false
```

Comment:
The last thing - could we add this to readme.documenation

And we should also pass the same option here: https://github.com/RedHatInsights/frontend-components/blob/266c00d4949a1664ef58732428021780a3059a8b/packages/config/src/config.js#L187

Reply:
I don't want to add this to docs yet. It's something that should only be used in chrome and nowhere else. It has the potential to break the env if used incorrectly. It would like to keep it :secret:.

Reply:
Okay, I get it.

So just please add it to the main config arguments. (Or does chrome use the proxy package directly?)

Reply:
Yeah it only uses the proxy. It has its own webpack config.

Reply:
Then we are ready to go 🚀 🎸